test/test_create_runtime_policy.py

```diff
@@ -797,206 +798,23 @@ def test_digest_algorithm_priority(self):
             expected_algo = case["expected_algo"]
             expected_source = case["expected_source"]
 
-            with self.assertLogs("policy.create_runtime_policy", level="DEBUG") as logs:
-                _policy = create_runtime_policy.create_runtime_policy(args)
-
-                self.assertIn(
-                    f"DEBUG:policy.create_runtime_policy:Using digest algorithm '{expected_algo}' obtained from the {expected_source}",
-                    logs.output,
-                )
-
-    def test_digest_algorithm_priority_exceptions(self):
```

Comment:
Apparently during rebases some tests were removed by accident. I think the removal of the `test_digest_algorithm_priority_exceptions()` was not intentional

Reply:
I believe I have resolved these issues; please double check.